schemaparser/schemajson_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schemaparser
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	testSchemaFile = "testdata/ext.json"
+)
+
+func TestLoadSchemaJSON(t *testing.T) {
+	sj, err := schemaJSON(testSchemaFile)
+	assert.Nil(t, err, "Failed to load schema into SchemaJSON struct.")
```

Comment:
Consider not using testify, which is a really problematic library IMO.  Error messages are not helpful, and the test suite proceeds even after one asserts something should be non-nil, but is. I've seen this in pipelines a lot, which is I why I'd prefer not to use this library at all. The Go team recommends writing table-driven tests instead:

https://github.com/golang/go/wiki/TableDrivenTests

Reply:
I have no preference and can use testing tables. Using it because it is shorter than writing `if err !=nil { t.Errorf(...)}` (You're right. here should be using t.Fatal)